common/v2/translations/lang/en.json

```diff
@@ -280,6 +280,7 @@
     "VIEWWALLET_SHOWPRIVKEY": "(show)",
     "VIEWWALLET_HIDEPRIVKEY": "(hide)",
     "CX_QUICKSEND": "QuickSend ",
+    "UNEXPECTED_ERROR": "Something went wrong. Please refresh this page and try again or contact us at $link",
     "ERROR_0": "Please enter a valid amount (Must be a number greater than or equal to 0) ",
     "ERROR_1": "Your password must be at least 9 characters. Please ensure it is a strong password. ",
     "ERROR_2": "Sorry! We don't recognize this type of wallet file. ",
```

Comment:
I would add a comma in "...try again, or contact..."
"Please enter a valid amount (must be a number greater than or equal to 0)."  - lowercase m and period at the end
The rest looks good to me, but is there supposed to be a space between the end of the sentence and the end quote on the last three?

Reply:
Actually what I was asking for was in `emailFormatter.ts`. If we want to define a certain message to include in the email generated for the user.